api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupMissedAppointmentList.java

```diff
@@ -115,7 +115,10 @@ public ReportDefinition constructReportDefinition() {
 
         PatientDataSetDefinition dsd = new PatientDataSetDefinition();
 
-        CohortDefinition definition = df.getMissedAppointment();
+        CohortDefinition deadPatients = df.getDeadPatientsDuringPeriod();
```

Comment:
@dbaluku is it possible to create a general cohort for people dead and transferred out so that we don't have to define it in every facility report but rather simply call it. It can be defined under cohortdefinitions since its going to be cutting across many report